input_format_test.go

```diff
@@ -0,0 +1,12 @@
+package astiav_test
+
+import (
+	"github.com/asticode/go-astiav"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestFindInputFormat(t *testing.T) {
```

Comment:
Could you:

- add both `func (f *InputFormat) Name() string` and `func (f *InputFormat) String() string` functions to the `InputFormat` struct, similar to the `Codec` struct?
- use the `rawvideo` input format instead in the test (otherwise test fails in windows or darwin)
- check first `inputFormat` is not nil and then use the `.Name()` function instead of the `.Flags()` function in the test